### PR TITLE
fix install path

### PR DESCRIPTION
**Pull request by denolf**
Fix to avoid that header files are installed in aiengine subfolder.

### CODE REVIEW

File: runtime_lib/xaiengine/CMakeLists.txt [hunk 1]
```diff
@@ -44,7 +44,11 @@ if (${CMAKE_CROSSCOMPILING})
 
 else()
     message("Copying xaiengine for ${AIE_RUNTIME_TARGET} from Vitis at ${VITIS_ROOT}.") 
-    set(xaiengineIncludePath ${VITIS_ROOT}/aietools/include/drivers/aiengine)
+    set(xaiengineIncludePath ${VITIS_AIETOOLS_DIR}/include/drivers/aiengine)
+    #tmp path to header files since we use an older (3_0) version of xaiengine. need to fix when we upgrade to 2023.1 and can take this from aie-rt
+    set(xaiengineIncludePathTmp ${VITIS_ROOT}/data/embeddedsw/XilinxProcessorIPLib/drivers/aienginev2_v3_0/src)
```
Comment (stephenneuendorffer) [inline runtime_lib/xaiengine/CMakeLists.txt:49]: I'd rather not proliferate this hard-coded path. We already have a problem that this version isn't consistent with the version used in aiesimulator.

Comment (denolf) [reply runtime_lib/xaiengine/CMakeLists.txt:55]: made path dependent on Vitis Version

File: runtime_lib/xaiengine/CMakeLists.txt [hunk 2]
```diff
@@ -6,15 +6,21 @@
...
 cmake_minimum_required(VERSION 3.20.1)
 
+#tmp path to header files since we use an older (3_0) version of xaiengine. need to fix when we upgrade to 2023.1
+if(${Vitis_VERSION_MAJOR} EQUAL "2022")
+    set(aieRTIncludePath "${VITIS_ROOT}/data/embeddedsw/XilinxProcessorIPLib/drivers/aienginev2_v3_0/src" CACHE STRING "AIE-RT include path")
+elseif(${Vitis_VERSION_MAJOR} EQUAL "2023")
+    set(aieRTIncludePath "${VITIS_AIETOOLS_DIR}/include/drivers/aiengine" CACHE STRING "AIE-RT include path")
+else()
+    message(FATAL_ERROR "Unsupported Vitis version: ${Vitis_VERSION_MAJOR}")
+endif()
```
Comment (jgmelber) [inline runtime_lib/xaiengine/CMakeLists.txt:16]: Good fix to ease the transition to aie-rt

**Review by jgmelber — APPROVED** [review]

**Comment by denolf** [comment]
merged #534

**Pull request merged**